app/database/operator/server_data_operator/transformers/user.ts

```diff
@@ -128,7 +128,7 @@ export const transformPreferenceRecord = ({action, database, value}: Transformer
  * @param {RecordPair} operator.value
  * @returns {Promise<ChannelMembershipModel>}
  */
-export const transformChannelMembershipRecord = ({action, database, value}: TransformerArgs): Promise<ChannelMembershipModel> => {
+export const transformMyChannelMembershipRecord = ({action, database, value}: TransformerArgs): Promise<ChannelMembershipModel> => {
```

Comment:
By definition, the ChannelMembership  represents the 'association table' where many channels have users and many users are on channels ( N:N relationship between model Users and model Channel).  Why are we changing this `transformChannelMembershipRecord` to `transformMyChannelMembershipRecord` ?  Will this now only hold association for the current user?

Reply:
The name is wrong. This was part of a refactoring I was trying, but it seems this particular step was completely wrong and was not properly reverted.